src/main/java/org/broadinstitute/hellbender/utils/gcs/BucketUtils.java

```diff
@@ -374,6 +374,17 @@ public static String getPathWithoutBucket(String path) {
 
     }
 
+    /**
+     * String -> Path. This *should* not be necessary (use Paths.get(URI.create(...)) instead) , but it currently is
+     * on Spark because using the fat, shaded jar breaks the registration of the GCS FilesystemProvider.
```

Comment:
I expect we can, yes. If nothing else by manually fixing META-INF/services.
Ideally this would be done automatically by the jar shader.